src/components/header/Header.jsx

```diff
@@ -107,6 +113,9 @@ export default function Header() {
               <div className={styles.basketIcon}>
                 <img src="/images/basket-default-noNotif.svg" alt="user icon" />
               </div>
+              <div>
+                <button onClick={() => handleOut()}>вийти</button>
```

Comment:
no need for  wrapper "handleOut" function. 
just call logout() from store directly.